src/core/evaluator.js

```diff
@@ -263,8 +263,14 @@ var PartialEvaluator = (function PartialEvaluatorClosure() {
           groupOptions.isolated = (group.get('I') || false);
           groupOptions.knockout = (group.get('K') || false);
           if (group.has('CS')) {
-            colorSpace = ColorSpace.parse(group.get('CS'), this.xref, resources,
-                                          this.pdfFunctionFactory);
+            colorSpace = group.get('CS');
+            if (colorSpace) {
+              colorSpace = ColorSpace.parse(colorSpace, this.xref, resources,
```

Comment:
Another possible solution would be use `try ... catch` instead, but given that similar bugs doesn't appear to be common that seems a bit unnecessary at the moment (also since it would probably be less efficient). In that case though, you'd probably only want to ignore errors if `this.options.ignoreErrors === true`.